newton/examples/example_quadruped.py

```diff
@@ -27,11 +27,12 @@
 import warp as wp
 
 import newton
-import newton.collision
 import newton.core.articulation
 import newton.examples
 import newton.utils
 
+wp.config.verify_cuda = True
+
 
 class Example:
     def __init__(self, stage_path="example_quadruped.usd", num_envs=8):
```

Comment:
I think we have to create a ground plane here now. We could also make the `builder._create_ground_plane()` function more public as it will be commonly used.